datachannel_test.go

```diff
@@ -223,39 +223,34 @@ func TestDataChannelParameters(t *testing.T) {
 	defer report()
 
 	t.Run("MaxPacketLifeTime exchange", func(t *testing.T) {
-		// Note(albrow): See https://github.com/node-webrtc/node-webrtc/issues/492.
-		// There is a bug in the npm wrtc package which causes this test to fail.
-		// TODO(albrow): Uncomment this once issue is resolved.
-		t.Skip("Skipping because of upstream issue")
-
-		// var ordered = true
-		// var maxPacketLifeTime uint16 = 3
-		// options := &DataChannelInit{
-		// 	Ordered:           &ordered,
-		// 	MaxPacketLifeTime: &maxPacketLifeTime,
-		// }
-
-		// offerPC, answerPC, dc, done := setUpReliabilityParamTest(t, options)
-
-		// // Check if parameters are correctly set
-		// assert.True(t, dc.Ordered(), "Ordered should be set to true")
-		// if assert.NotNil(t, dc.MaxPacketLifeTime(), "should not be nil") {
-		// 	assert.Equal(t, maxPacketLifeTime, *dc.MaxPacketLifeTime(), "should match")
-		// }
-
-		// answerPC.OnDataChannel(func(d *DataChannel) {
-		// 	if d.Label() != expectedLabel {
-		// 		return
-		// 	}
-		// 	// Check if parameters are correctly set
-		// 	assert.True(t, d.Ordered(), "Ordered should be set to true")
-		// 	if assert.NotNil(t, d.MaxPacketLifeTime(), "should not be nil") {
-		// 		assert.Equal(t, maxPacketLifeTime, *d.MaxPacketLifeTime(), "should match")
-		// 	}
-		// 	done <- true
-		// })
-
-		// closeReliabilityParamTest(t, offerPC, answerPC, done)
+		var ordered = true
```

Comment:
Maybe this is just a stylistic source but I see no reason not to use the `:=` shorthand here.

```suggestion
		ordered := true
```

Reply:
agree, fixed! The `var` is just extra to distract the reader, and offers no real value :)